frontend/app/templates/components/forms/signup-form.hbs

```diff
@@ -1,48 +1,54 @@
 <div class="ui stackable three column doubling centered grid signup">
-<div class="column">
-    <form class="ui form" onsubmit={{action 'signUp'}}>
-        <div class="ui aligned segment basic form-div">
-          <span class="form-title">Registration Form</span>
-          <div></div>
-          <span class="form-subtitle">Sign up today</span>
-        </div>
-        <div class="ui stacked element">
-            <div class="field required">
-                <div class="ui left icon input">
-                    <i class="mail icon"></i>
-                    {{input type="text" value=email name="email" placeholder="E-mail address"}}
-                </div>
+    <div class="column">
+        <form class="ui form" onsubmit={{action 'signUp'}}>
+            <div class="ui aligned segment basic form-div">
+                <span class="form-title">Registration Form</span>
+                <div></div>
+                <span class="form-subtitle">Sign up today</span>
             </div>
-            <div class="field required">
-                <div class="ui left icon input">
-                    <i class="user icon"></i>
-                    {{input type="text" value=username name="username" placeholder="Username"}}
+            <div class="ui stacked element">
+                <div class="field required">
+                    <div class="ui left icon input">
+                        <i class="mail icon"></i>
+                        {{input type="text" value=email name="email" placeholder="E-mail address"}}
+                    </div>
                 </div>
-            </div>
-            <div class="field required">
-                <div class="ui left icon input">
-                    <i class="lock icon"></i>
-                    {{input type="password" value=password name="password" placeholder="Password"}}
+                <div class="field required">
+                    <div class="ui left icon input">
+                        <i class="user icon"></i>
+                        {{input type="text" value=username name="username" placeholder="Username"}}
+                    </div>
                 </div>
-            </div>
-            <div class="field required">
-                <div class="ui left icon input">
-                    <i class="lock icon"></i>
-                    {{input type="password" name="password_repeat" placeholder="Re-enter Password"}}
+                <div class="field required">
+                    <div class="ui left icon input">
+                        <i class="lock icon"></i>
+                        {{input type="password" id="pwd1" value=password name="password" placeholder="Password"}}
+                        <button type="button" id="eye1" {{action 'show1'}}>
+                            <img src="https://raw.githubusercontent.com/fossasia/badgeyay/development/frontend/public/images/eye-16.png" alt="eye">
```

Comment:
@rajvaibhavdubey Please refer it as `https://github.com/fossasia/badgeyay/development/frontend/public/images/eye-16.png`